docs/source/adapters/site.rst

```diff
@@ -527,17 +527,25 @@ Available machine type configuration options
 
 .. content-tabs:: left-col
 
-    +----------------+--------------------------------------------------------------------------+-----------------+
-    | Option         | Short Description                                                        | Requirement     |
-    +================+==========================================================================+=================+
-    | namespace      | Namespace for pods to run in.                                            |  **Required**   |
-    +----------------+--------------------------------------------------------------------------+-----------------+
-    | image          | Image for the pods.                                                      |  **Required**   |
-    +----------------+--------------------------------------------------------------------------+-----------------+
-    | label          | Label to be assigned to your pods.                                       |  **Required**   |
-    +----------------+--------------------------------------------------------------------------+-----------------+
-    | args           | Arguments for the containers that run in your pods .                     |  **Optional**   |
-    +----------------+--------------------------------------------------------------------------+-----------------+
+    +----------------+-------------------------------------------------------------------------------+-----------------+
+    | Option         | Short Description                                                             | Requirement     |
+    +================+===============================================================================+=================+
+    | namespace      | Namespace for pods to run in.                                                 |  **Required**   |
+    +----------------+-------------------------------------------------------------------------------+-----------------+
+    | image          | Image for the pods.                                                           |  **Required**   |
+    +----------------+-------------------------------------------------------------------------------+-----------------+
+    | args           | Arguments for the containers that run in your pods.                           |  **Required**   |
```

Comment:
Why are `args` now required?

Reply:
They were required in the previous version as well, but it was wrongly stated.

Reply:
Okay!